src/canisters/governance/response.converters.ts

```diff
@@ -767,3 +769,10 @@ export const toMakeProposalResponse = (
   }
   throw throwUnrecognisedTypeError({ name: "response", value: response });
 };
+
+export const toGovernanceError = (
```

Comment:
- do we actually need to parse the error - i.e. do we do that in other functions of nns-js?
if yes, of course we follow the pattern but asking before yesterday I noticed we don't within the nns-dapp and the nns-dapp canister

- code: can be destructed

```
toGovernanceError = ({error_message: errorMessage, .... => ({errorMessage, ...
```

Reply:
I wanted to keep the same types to do that I needed to convert it.

I think that this is a good and broad question. More towards how we want to keep develping nns-js.

Reply:
just opened the project, indeed that's the nns-js pattern. let's keep it as it, you are right.

maybe just the "deconstructing" part for the state of the art? 😉

Reply:
that part done 🙂